shared/macros.jinja

```diff
@@ -1745,3 +1745,44 @@ Fill in <tt>GRUBENV_FILE_LOCATION</tt> based on information above.
 
     $ systemctl enable --now {{{ service }}}
 {{%- endmacro -%}}
+
+{{#
```

Comment:
attribute `value` is not being used in `kernel_build_config_describe_config` and `kernel_build_config_ocil`. The macros can be clever by using `value` to decide if the configuration should be present or not in the boot config file. Even better if a different attribute is used to decide if the line should be absent or not similarly as suggested in the template data.